schemas/v2.0.0.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  version: '2.0.0',
+  baseUrl: 'https://prod-1-dc-api-oopeix3r.deepcrawl.com',
```

Comment:
It won't even be the same for **us** since we know it's going to change before this goes live. When you already know something will be variable like that, it belongs in config. Also consider that different versions of the schema (2.0.1, 2.1.0, etc.) will almost certainly use the same base url, and so you wouldn't want that duplicated. You'd want it configured at the top level and passed into the API code.